docs/herramientas_test.md

```diff
@@ -0,0 +1,43 @@
+# Herramientas de test
+
+## Criterios de decisión
+
+Para elegir las diferentes herramientas para nuestro proyecto tendremos que tener en cuenta una seria de criterios que seran los siguientes:
+
+- **Compatibilidad**: La herramienta debe ser compatible con el lenguaje de programación que estamos utilizando en nuestro caso TypeScript.
+
+- **Comunidad**: La herramienta debe tener una comunidad que pueda ayudar a resolver problemas.
+
+- **Híbrido**: Se tendrá en cuenta que la herramienta tenga biblioteca para aserciones así como el test runner.
+
+
+## Herramientas de test
+
+Necesitamos elegir herramientas de test para nuestro proyecto. Para ello será necesario elegir un una biblioteca de aserciones, un test runner y un CLI para ejecutar los test.
+
+### Aserciones
+
+- [Chai](https://www.chaijs.com/): Chai es una biblioteca de aserciones para Node.js y el navegador. Chai tiene una sintaxis legible y es fácil de usar.
+
+- [Jest](https://jestjs.io/): Jest es realmente un framework, sin embargo viene incluido con una biblioteca de aserciones.
+
+- [Should.js](https://shouldjs.github.io/): Should.js es una biblioteca de aserciones para Node.js y el navegador. Should.js tiene una sintaxis limpia y fácil de usar. Sin embargo no tienen una comunidad tan grande como otras bibliotecas de aserciones.
```

Comment:
Por favor, aprended a usar sistemáticamente [Snyk advisor](https://snyk.io/advisor/npm-package/should). Da una puntuación numérica que te puede ayudar a tomar decisiones.

Reply:
Como has mencionado he usado Snyk advisor como uno de los criterios. Solucionado en bfd66992b67723a0e993eeeda3045de8899dbc06